tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature

```diff
@@ -219,7 +219,7 @@ Feature: Share by public link with different roles
     And the user uploads folder "PARENT" using the webUI
     Then folder "PARENT" should be listed on the webUI
     And folder "CHILD" should be listed in the folder "PARENT" on the webUI
-    And file "child.txt" should be listed in the folder "CHILD" on the webUI
+    And file "child.txt" should be listed in the folder "CHILD" on the webUI as single share
```

Comment:
@kulmann not sure if expected behavior product-wise

Reply:
As in "you're a user on a public link, create a folder, navigate into folder, create/upload a file and see this file as a single share (and not in a table view)"